pkg/controlplane/instance_openapi_test.go

```diff
@@ -51,6 +52,7 @@ func TestValidOpenAPISpec(t *testing.T) {
 			Version: "unversioned",
 		},
 	}
+	config.ExtraConfig.ServiceIPRange = net.IPNet{IP: net.ParseIP("10.0.0.0"), Mask: net.CIDRMask(24, 32)}
```

Comment:
This should be unnecessary given that you modified the `setUp` function below to populate the ServiceIPRange right?

Reply:
Thanks! I fixed it.